Packs/ZeroTrustAnalyticsPlatform/Playbooks/playbook-ZTAP_Alert_README.md

```diff
@@ -0,0 +1,39 @@
+This playbok is triggered by fetching escalated ZTAP Alerts.
+The playbook fetches newly escalated alerts.
+Then, the playbook performs enrichment on the incident's indicators.
+Lastly, it adds comments/logs as Evidence.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* Calculate Severity - Generic v2
```

Comment:
Please generate new README file after the playbook modifications

Reply:
And add new picture.